tests/ignite/contrib/metrics/regression/test_manhattan_distance.py

```diff
@@ -0,0 +1,38 @@
+from ignite.exceptions import NotComputableError
+from ignite.contrib.metrics.regression import ManhattanDistance
+import torch
+import numpy as np
+import pytest
+
+
+def test_zero_div():
```

Comment:
There is no div at all in `ManhattanDistance`, so this shouldn't be called `test_zero_div`

Reply:
Please see comment above about removing NotComputableError entirely.